index.js

```diff
@@ -211,6 +217,7 @@ class TagInput<T> extends React.PureComponent<Props<T>, State> {
     if (this.props.text !== '' || event.nativeEvent.key !== 'Backspace') {
       return;
     }
+    return;
```

Comment:
What's this for?

Reply:
Sorry for that, I forgot it. I put it just for my code..